rpc/transport/http/client.go

```diff
@@ -29,10 +30,11 @@ func NewHttpTransportAsClient(url string, retryTimeout time.Duration) (*clientHt
 		return nil, err
 	}
 
-	subscribeUrl, err := urlUtil.JoinPath("ws://", url, "subscribe")
+	subscribeUrl, err := urlUtil.JoinPath("wss://", url, "subscribe")
```

Comment:
Right now the `start-rpc-servers` doesn't have specify a cert file, so will run over http/ws. This means that the rpc client won't be able to connect to any of the clients since it assumes https/wss.

We could probably update our CLI to take a url with a protocol(`http://localhost:4005/v1/subscribe` instead of `localhost:4005/v1/subscribe`).   `http` would mean `http/ws` and `https` would mean `https/wss`, that way the rpc clients could support either.

Reply:
Check out https://github.com/statechannels/go-nitro/blob/327591540a862741e7fb4fe1e448f7dc57ca6d4d/tls/readme.md?plain=1#L9. Does that help?

Note that all of the tests currently pass, including the create channels Typescript test.

Reply:
Hmm I'm still seeing the same issue.
From the root `go-nitro` directory I ran:
- `make -C tls create-cert`
- ` go run ./cmd/start-rpc-servers/main.go`
-`npx ts-node ./packages/nitro-rpc-client/scripts/client-runner.ts create-channels`

Reply:
Oops I missed running `export NODE_EXTRA_CA_CERTS="$(mkcert -CAROOT)/rootCA.pem". That resolved it!